dts/bindings/i3c/nxp,mcux-i3c.yaml

```diff
@@ -24,18 +24,25 @@ properties:
 
   clk-divider:
     type: int
-    description: Main clock divider for I3C
-    required: true
+    default: 0
+    description: |
+      Main clock divider for I3C. If not specified, the default value
+      will be used if changing clock divider is supported by the SoC.
 
   clk-divider-tc:
     type: int
-    description: TC clock divider for I3C
-    required: true
+    default: 0
```

Comment:
Why not do the same as above i.e use `DT_INST_PROP_OR` in the driver and leave this untouched.

Reply:
I've updated